ts/src/vendor-specific-values.ts

```diff
@@ -3,6 +3,7 @@ import { SourceType } from './source-type'
 export type VendorSpecificValues = {
   maxEventLevelChannelCapacityPerSource: Record<SourceType, number>
   maxSettableEventLevelEpsilon: number
+  maxCardinalityOfPossibleTriggerStates: number
```

Comment:
Let's rename this field to match the spec, i.e. `maxTriggerStateCardinality`.

Reply:
Done!